stakepooldclient/stakepooldclient.go

```diff
@@ -498,3 +498,32 @@ func (s *StakepooldManager) RPCStatus() []string {
 
 	return stakepooldPageInfo
 }
+
+// MyMasterPubKeys performs gRPC MasterPubKeys requests against all stakepoold
+// instances and returns all results fetched without errors. Returns a map of keys
```

Comment:
Comment suggests "returns without errors" but there are several places in the code below which return errors.

I would suggest that we return an error on any failure, or any on mis-matched results. All of the hot wallets should be set up with the same seed, and so they should have the same accounts and master pubkeys. Any mismatch indicates that the VSP operator has to take some remedial action

Reply:
Ok!

Reply:
ok. This allows for different names, but not for different keys with the same name.  I think this is enough, unless really think they should be exact.  The accounts for my pools have different names from testing now... As long as default is the same I think the pool will function as intended. Even changing the default should not cause a problem, as long as both are changed.

Reply:
Copying my comment from Matrix, to ensure I dont forget about it:

> I think we can simplify things here, I dont think we need to compare all of the master pub keys. We can just get:
>
> - pool1keys["default"]
> - pool2keys["default"]
> - pool3keys["default"]
>
> If they are all equal, return that value. If any of them are different, return an error.
> I dont think we need to worry about people renaming the default account or otherwise customising the voting wallets, these wallets should be used for a single purpose - voting for the VSP. If operators want to mess around on their wallets and do things outside of the prescribed process, they can deal with that extra complexity themselves.